SlevomatCodingStandard/Helpers/AnnotationTypeHelper.php

```diff
@@ -85,7 +85,7 @@ public static function getIdentifierTypeNodes(TypeNode $typeNode): array
 		}
 
 		/** @var IdentifierTypeNode|ThisTypeNode $typeNode */
-		$typeNode = $typeNode;
+		$typeNode = clone $typeNode;
```

Comment:
This commit is completly wrong. You should ignore these assignments in the sniff - ignore when there is comment with `@var` annotations before.

Reply:
@kukulich `@codeCoverageIgnoreStart` and `@codeCoverageIgnoreEnd`?
Completly? What's about?
~~~php
$semicolonPointer = null;
...
/** @var int $semicolonPointer */
$semicolonPointer = (int) $semicolonPointer;
~~~

Reply:
No, I don't want `@codeCoverageIgnoreStart`

The `DuplicateAssignmentToVariableSniff` should not report these lines at all.

Reply:
@kukulich my PR add detecting definations as `$semicolonPointer = $semicolonPointer;` and it's failed current tests https://github.com/slevomat/coding-standard/runs/703302581#step:7:17

Reply:
The PR is still wrong.

Reply:
@kukulich but problem not in my changes - https://github.com/slevomat/coding-standard/pull/1017/checks?check_run_id=703433286#step:7:16

Reply:
@wietsewarendorff the change you made in DuplicateAssignmentToVariableSniff is not right, see my previous comments

Reply:
I guess you tried to mention the topic starter here? :)

Reply:
@wietsewarendorff Sorry, stupid github on mobile :)